WordPress/WordPressTest/Dashboard/DashboardPostsSyncManagerTests.swift

```diff
@@ -179,14 +183,13 @@ class DashboardPostsSyncManagerTests: CoreDataTestCase {
         XCTAssertEqual(listener.postsSyncType, .post)
         XCTAssertEqual(blog.dashboardState.postsSyncingStatuses, [])
         XCTAssertTrue(blogService.syncAuthorsCalled)
-        XCTAssertFalse(postService.syncPostsCalled)
     }
 
 }
 
-class SyncManagerListenerMock: DashboardPostsSyncManagerListener {
+class SyncManagerListenerMock: NSObject, DashboardPostsSyncManagerListener {
 
-    var postsSyncedCalled = false
+    @objc dynamic var postsSyncedCalled = false
```

Comment:
What do you think of having these properties `private(set)`?

I like if from a safety point of view, just to make sure that a test doesn't accidentally update them directly, as opposed to the side effect of the SUT interacting with the test double.

In practice, I acknowledge it's much of a muchness and requires extra typing (unless you get AI to help, but that's hit and miss).

Reply:
Sounds reasonable. These properties are meant to be private(set) after all. I doubt they were intentionally declared as allowing other to change their value.